test/e2e_node/gpu_device_plugin.go

```diff
@@ -89,24 +85,28 @@ var _ = framework.KubeDescribe("NVIDIA GPU Device Plugin [Feature:GPUDevicePlugi
 
 		It("checks that when Kubelet restarts exclusive GPU assignation to pods is kept.", func() {
 			By("Creating one GPU pod on a node with at least two GPUs")
-			p1 := f.PodClient().CreateSync(makeCudaPauseImage())
-			count1, devId1 := getDeviceId(f, p1.Name, p1.Name, 1)
+			podRECMD := "devs=$(ls /dev/ | egrep '^nvidia[0-9]+$') && echo gpu devices: $devs"
```

Comment:
Let's continue testing GPU images with NVIDIA images.

Reply:
I think this is out of this PR's scope, here we just reuse the same function :)

Reply:
No I don't think it's out of scope to keep the same behavior.
If you don't test GPU images we might as well not call this test `GPU_device_plugin`

Reply:
This file `gpu_device_plugin.go` is created in another PR, maybe we should discuss this in a new thread.

Reply:
Looking a bit more at the code you are right, the current image is a busybox image.
Not sure what's the point of testing that and still call this a GPU image.